static/views/property-meta.php

```diff
@@ -11,7 +11,12 @@
     $meta_value = do_shortcode(html_entity_decode(get_post_meta( $post_id, $meta_slug, true )));
     if(trim($meta_value) != ""):
```

Comment:
@andypotanin Not sure about this "if" block. [not added by me].
To play it safe I have added "isset" for now.